standard_releases.md

```diff
@@ -1,59 +1,206 @@
+* Review, test and merge those changes into the `master` branch.
+* Depending on the other ongoing work, either call a "freeze" on the master
+  branch, whereafter merging changes that are not going to be part of the
+  release cannot be merged, or create a new release branch from master. A
+  release freeze is preferred, to reduce the complexity of incorporating any
+  release fixes back into master, but this can be weighed against the impediment
+  to other development on a case-by-case basis.
+* Begin translation of the new release (see [translations](/translations)) and
+  fixing of any snags or bugs. If `master` is frozen, these changes can be merged
+  into master, if using a release branch, merge them there for incorporation
+  into master after the release.
+* Once the release is finalised and everything is ready, release the new version
+  on ReadTheDocs, either by making a new release branch, or simply enabling the
+  existing one. Remember to update the 'latest' branch pointer, and update other
+  release branches to point to it.
+* If fixes and snagging were done on a release branch, now is the time to merge
+  that branch back into master and fix any conflicts with what's been
+  merged in the meantime.
+
+### A new language to the current release
+
+Until such time that we support several stable releases, our assumption is that
+only the current release (and any future ones) will be translated into new
+languages.
+
+When there are no significant differences between `master` and the current
+release, the process should be to translate master and then copy those
+translations over to the current release 'project' in Transifex.
+
+* Make a new branch from `master` for the translation work
+* Review, test, snag and finally merge this branch into `master`
+* Make a new branch from the current release and bring across the new
+  translations from Transifex.
+* As before, review this branch and fix any snagging issues with the translation
+  in this context. It is assumed that normally these fixes will not need to be
+  merged back into master, but if they uncover bugs not previously found, those
+  should be `git cherry-pick`-ed back into `master`.
+
+When `master` and the current release have diverged significantly (a situation
+we've yet to encounter), it's assumed that the correct process will be to treat
+the translation of each as a separate project. In this case, it's probably most
+helpful to translate the current release first, bring across any fixes and only
+initiate translation of `master` when it's due for release.
+
+### Fixes to an existing release
+
+When a bug is discovered or a correction needs to be made to an existing
+release and it is deemed severe enough to require an immediate fix (i.e.
+it cannot wait for the next naturally-occurring release), then we need
+to make an unplanned release. The process for this is as follows:
+
+* Make a new branch named something like `v0-2-0-description-of-issue`
```

Comment:
An alternative would be to raise an issue and use the issue number as the reference. I think this has been the way we have tended to name branches that fix issues in the past (mea culpa, I've also failed to use either method on occasion)

Reply:
Good point - I've added the github issue based process in dddcbfb.